ipaserver/install/ipa_migrate.py

```diff
@@ -761,13 +760,14 @@ def connect_to_remote_ds(self):
         insecure_bind = False
 
         if self.args.cacertfile is not None:
-            # Store CA cert into file
-            tmp_ca_cert_f = write_tmp_file(self.args.cacertfile)
-            cacert = tmp_ca_cert_f.name
-
             # Start TLS connection (START_TLS)
-            ds_conn = LDAPClient(ldapuri, cacert=cacert, start_tls=True)
-            tmp_ca_cert_f.close()
+            try:
+                ds_conn = LDAPClient(ldapuri, cacert=self.args.cacertfile,
+                                     start_tls=True)
+            except (ldap.LDAPError, errors.NetworkError) as e:
```

Comment:
This also needs to handle errors.DatabaseError otherwise you get a traceback if you pass in the wrong CA chain.

It also needs to catch IOError if a non-existent file is referenced.

Reply:
> This also needs to handle errors.DatabaseError otherwise you get a traceback if you pass in the wrong CA chain.
> 
> It also needs to catch IOError if a non-existent file is referenced.

Fixed